nfv_tempest_plugin/tests/scenario/test_nfv_sriov_usecases.py

```diff
@@ -512,3 +513,73 @@ def test_sriov_min_qos(self, test='min_qos'):
         self.run_iperf_test(qos_policies, servers, key_pair, net_id)
         LOG.info('SRIOV Min QoS test, check test result.')
         self.collect_iperf_results(qos_rules_list, servers, key_pair)
+
+    def test_sriov_dcb_config(self, test='min_qos'):
+        """Test SRIOV DCB Config
+
+        SUPPORTED: Mellanox NICS only
+        """
+
+        LOG.info('Start SRIOV DCB Config test, search for Mellanox nics')
+        # Check setup contains Mellanox nics.
+        kw_args = dict()
+        kw_args['command'] = r"sudo lshw -class network -businfo | grep "
+        kw_args['file_path'] = CONF.nfv_plugin_options.conf_files['sriov-nova']
+        kw_args['search_param'] = \
+            {'section': 'pci', 'value': 'device_spec'}
+        """ Regexp search Mellanox connect-x """
+        kw_args['filter_regexp'] = \
+            r'.*\[ConnectX\-5 Ex\]|.*\[ConnectX\-5\]'
```

Comment:
Only connectx5?

Reply:
Yes. I will add ConnectX-6 as well. But currently we have an issue with ConnectX-6 and I have opened a JIRA for the same.

Reply:
Changes updated. Kindly check.